sphinx_gallery/tests/test_gen_rst.py

```diff
@@ -547,7 +548,21 @@ def test_rst_example(gallery_conf):
     assert "rst-class:: sphx-glr-timing" in rst
 
 
-def test_output_indentation(gallery_conf, tmpdir):
+@pytest.fixture(scope='function')
+def script_vars(tmpdir):
+    fake_main = imp.new_module('__main__')
```

Comment:
```suggestion
    fake_main = importlib.util.module_from_spec(
        importlib.util.spec_from_loader('__main__'))
```

Reply:
Not sure if `None` is needed in `importlib.util.spec_from_loader('__main__')`

`importlib` is much more verbose!